browser-switch/build.gradle

```diff
@@ -77,76 +78,73 @@ artifacts {
-                    authentication(userName: sonatypeUsername, password: sonatypePassword)
-                }
-            }
+signing {
+    required {
+        !version.endsWith("DEVELOPMENT")
+    }
+    sign publishing.publications
+}
 
-            pom.setArtifactId 'browser-switch'
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+                // Applies the component for the release build variant.
+                from components.release
 
-            pom.project {
-                name 'browser-switch'
+                // You can then customize attributes of the publication as shown below.
                 groupId = 'com.braintreepayments'
+                artifactId = 'browser-switch'
                 version = "${android.defaultConfig.versionName}"
-                packaging 'aar'
-                description 'A library making it easy to open a url in a browser or Chrome Custom Tab and receive a response from the web page.'
 
-                url 'https://github.com/braintree/browser-switch-android'
+                pom {
+                    name = 'browser-switch'
+                    packaging = 'aar'
+                    description = 'A library making it easy to open a url in a browser or Chrome Custom Tab and receive a response from the web page.'
+                    url = 'https://github.com/braintree/browser-switch-android'
 
-                scm {
-                    url 'scm:git@github.com:braintree/browser-switch-android.git'
-                    connection 'scm:git@github.com:braintree/browser-switch-android.git'
-                    developerConnection 'scm:git@github.com:braintree/browser-switch-android.git'
-                }
-
-                developers {
-                    developer {
-                        id 'devs'
-                        name 'Braintree Payments'
+                    scm {
+                        url = 'scm:git@github.com:braintree/browser-switch-android.git'
+                        connection = 'scm:git@github.com:braintree/browser-switch-android.git'
+                        developerConnection = 'scm:git@github.com:braintree/browser-switch-android.git'
                     }
-                }
 
-                licenses {
-                    license {
-                        name 'MIT'
-                        url 'http://opensource.org/licenses/MIT'
-                        distribution 'repo'
+                    developers {
+                        developer {
+                            id = 'devs'
+                            name = 'Braintree Payments'
+                        }
                     }
-                }
-            }
 
-            pom.whenConfigured { MavenPom pom ->
-                pom.dependencies.each {
-                    if (it.artifactId == 'support-annotations') {
-                        it.optional = true
+                    licenses {
```

Comment:
Maybe its a good time to update the dates in https://github.com/braintree/browser-switch-android/blob/master/LICENSE

Reply:
Def.